### PR TITLE
Where them peaches at? // a.k.a Service Cyborg botany tools!

### PR DESCRIPTION
**Pull request by Aliceee2ch**
## About The Pull Request
(my dumb ass commited previous pr on master branch so i made new 💀)

- Adds "Service Cyborg Botany Upgrade" that cost 13 iron and 2 glass.
- Basically, the mostly unused cyborg model has now more functionality. (as we know how many gimmicks can be made through botany)

- Botany Omni-tool:
gorg_newskin

- Meet the "Gardener"
gorg_gardener

## Why It's Good For The Game

I was outraged that despite the borg ability to cook - near half of the recipes you could not recreate due to the lack of a botanist (or when theyre doing syndicate operations). The Service borg can now serve well on a lowpop rounds.

## Changelog

:cl:
add: Added new botany module in mechfabricator designs. Gives service cyborg plant bag and plant analyzer.
add: Added cyborg botanical omni-tool.
qol: Service apparatus (another existing module) can now hold: seeds, fish, grafts.
qol: Secondary beaker storage can now hold any beakers. It used to spawn with one, but when you dropped it - you could no longer pick it back.
image: New gardener skin for service cyborg.

/:cl:

### CODE REVIEW

**Comment by AtomTheProphet** [comment]
Maybe could we get some sort of water synthesizing watering can, so the borgs can water the plants?

**Comment by Aliceee2ch** [comment]
> Would be nice if they had a watering can, but otherwise great.

no need, they already have beaker and droppers

**Comment by AtomTheProphet** [comment]
> > Would be nice if they had a watering can, but otherwise great.
> 
> no need, they already have beaker and droppers

It seems like it would be tedious to constantly switch between beakers for fertilizer and water. Also droppers don't transfer enough reagents for proper large volume watering.

**Comment by excalisword** [comment]
You could also just make it so the beaker apparatus's can pick up watering cans.

**Comment by Aliceee2ch** [comment]
> You could also just make it so the beaker apparatus's can pick up watering cans.

nah, thats weird. beaker should be for beakers only. watering cans - ill consider this maybe

**Comment by excalisword** [comment]
> nah, thats weird. beaker should be for beakers only. watering cans - ill consider this maybe

You could make the service apparatus able to pick up watering cans.

**Review by Wallemations — CHANGES_REQUESTED** [review]
The hatchet sprite's animation seems way too jittery imo, and it happens way too frequently.
Is it missing an in-between for going between its resting and upright states?

Also add a few extra ms of delay to the resting frame
The rest feels fine? Maybe just the cultivator also looking a little jittery but that's more another frame delay issue rather than needing a whole extra in-between.

Admittedly I'm only able to look at the gif in the body of the PR because I'm on mobile, so it's a little hard to see the small sprites, but I'm not about to make you open up your image editor and export a bunch of enlarged gifs for me so I'll check to see icondiff tomorrow to give better advice if needed.

File: code/__DEFINES/tools.dm [hunk 1]
```diff
@@ -21,6 +21,10 @@
 #define TOOL_ROLLINGPIN "rolling pin"
 /// Can be used to scrape rust off an any atom; which will result in the Rust Component being qdel'd
 #define TOOL_RUSTSCRAPER "rustscraper"
+/// Currently used only in borg botany omnitool, you may add some funny interaction with it later
+#define TOOL_CULTIVATOR "cultivator"
+#define TOOL_SECATEUR "secateur"
+#define TOOL_HATCHET "hatchet"
```
Comment (SmArtKar) [inline code/__DEFINES/tools.dm:27]: This is really bad, you shouldn't be adding tool types purely for the sake of making poorly written code (omnitools) work

**Comment by Aliceee2ch** [comment]
- Hatcher animation is better now.
hatchet

**Comment by Dawnseer** [comment]
I have a solution to the 'watering can' issue. Put a watering can in the botany omni tool.

**Comment by Dawnseer** [comment]
A thing to keep in mind, borgs can't really get stuff from vendors, so you'll have a hard time getting seeds to begin with.

**Comment by Aliceee2ch** [comment]
> A thing to keep in mind, borgs can't really get stuff from vendors, so you'll have a hard time getting seeds to begin with.

most (if not 90% seeds) can be either bought from cargo (vegetables, seeds or fruit crates), guaranteed to be found in: permabrig/mining station/gulag hydroponics, or in most gateways (theres atleast 2 or 3 gateway with seed vending machine that sells seeds for free).

**Comment by Dawnseer** [comment]
If you steal the perma seed crate then only hell awaits

**Review by Wallemations — APPROVED** [review]
I forgot to approve this
Still needs code maint approval though

File: code/modules/mining/equipment/mining_tools.dm [hunk 2]
```diff
@@ -163,6 +163,17 @@
...
+	name = "cyborg spade"
+	icon = 'icons/obj/items_cyborg.dmi'
+	icon_state = "sili_shovel"
+	toolspeed = 0.6
+	worn_icon_state = null
+
+/obj/item/shovel/spade/cyborg/Initialize(mapload)
+	. = ..()
+	ADD_TRAIT(src, TRAIT_NODROP, CYBORG_ITEM_TRAIT)
+
```
Comment (Ghommie) [inline code/modules/mining/equipment/mining_tools.dm:176]:
```suggestion
```

**Review by Ghommie — CHANGES_REQUESTED** [review]

**Review by Ghommie — APPROVED** [review]

**Pull request merged**